meta-mender-core/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -1,4 +1,4 @@
-PACKAGECONFIG += "networkd resolved"
+PACKAGECONFIG_append = "networkd resolved"
```

Comment:
I was wondering why you had to mod systemd config also. It would be nice to minimize changes to recipes like this.